internal/services/video_service_test.go

```diff
@@ -17,7 +17,7 @@ import (
 func init() {
 	err := godotenv.Load("../../.env")
```

Comment:
I know it's not part of this PR, but come on, this path can be passed by flag or existiur in the current folder